templates/fedora.tpl.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     template.kubevirt.io/version: v1alpha1
     defaults.template.kubevirt.io/disk: rootdisk
     template.kubevirt.io/containerdisks: |
-      quay.io/containerdisks/fedora:35
+      quay.io/containerdisks/fedora:36
```

Comment:
@dominikholler is it ok to keep only 36, or do we want to keep both?

Reply:
Let's go with only 36 for  now, to be sure the version 36 is shown in UI.

@upalatucci if there would be multiple containerdisks, which one would be shown in the UI?